### DISCUSSION TITLE
Setting up airbyte ETL on AWS fails

### DISCUSSION THREAD

**Discussion by anmol1vw13** [airbytehq/airbyte#38857] [Connector Questions] [answered]
I have an ec2 instance where I'm running airbyte (in a dockerized environment). 
I've set up a source(postgres) and a destination (snowflake). But when I try adding the destination to source it fails while reading the schema.
The stacktrace says: 
<img width="1512" alt="Screenshot 2024-06-03 at 14 25 05" src="https://github.com/airbytehq/airbyte/assets/13599208/50f48754-4a6b-4a82-b706-dfe0707eb8ec">

I don't see a fix even after going through the codebase. 
One of the issues suggest to set global.airbyteUrl, but I'm clear on where to set this.
https://github.com/airbytehq/airbyte/issues/38854

**Comment by rcourivaud** [comment]
I've the same error on GCP deployment ! I'm currently trying to downgrade the version to 0.60.1. I'll keep you in touch !

**Reply by anmol1vw13** [reply]
Thanks @rcourivaud

**Comment by anmol1vw13** [comment, marked as answer]
Downgrading to 0.60.1 worked

[a comment by an automated account is not shown]
